config/locales/en.yml

```diff
@@ -182,6 +182,9 @@ en:
       follow_up_patients_card:
```

Comment:
What's the difference in usage between `follow_up_patients_card` and `follow_up_visits_card`?

Reply:
In yearly reports, we add up the number of monthly reports. The aggregate number now indicates follow visits made at that facility and not the unique patients visited at that facility.
The same id true for monthly reports as well.
More details here
https://simpledotorg.slack.com/archives/C0190BDFRK2/p1664967435685769?thread_ts=1664887896.482949&cid=C0190BDFRK2

Reply:
Aren't monthly reports are counting unique patient visits on a monthly visit? The query does a distinct on `patient_id` for each month https://github.com/simpledotorg/simple-server/blob/35b42b06bc6ca53ecc3bd8873a240b8c92076fa0/db/views/reporting_facility_monthly_follow_ups_and_registrations_v01.sql#L57